docs/content/toc.md

```diff
@@ -126,20 +132,19 @@ layout: toc
   * [Metrics and Monitoring](/docs/VERSION/operations/metrics.html)
   * [Alerts](/docs/VERSION/operations/alerts.html)
   * [Different Hadoop Versions](/docs/VERSION/operations/other-hadoop.html)
-  * [HTTP Compression](/docs/VERSION/operations/http-compression.html)
-  * [Basic Cluster Tuning](/docs/VERSION/operations/basic-cluster-tuning.html)
-  * Examples
-      * [Single-server Deployment Examples](/docs/VERSION/operations/single-server.html)
-      * [Clustered Deployment Example](/docs/VERSION/operations/example-cluster.html)
-  * [Recommendations](/docs/VERSION/operations/recommendations.html)
-    * [Performance FAQ](/docs/VERSION/operations/performance-faq.html)
+  * [HTTP Compression](/docs/VERSION/operations/http-compression.html)  
   * [API Reference](/docs/VERSION/operations/api-reference.html)
       * [Coordinator](/docs/VERSION/operations/api-reference.html#coordinator)
       * [Overlord](/docs/VERSION/operations/api-reference.html#overlord)
       * [MiddleManager](/docs/VERSION/operations/api-reference.html#middlemanager)
       * [Peon](/docs/VERSION/operations/api-reference.html#peon)
       * [Broker](/docs/VERSION/operations/api-reference.html#broker)
-      * [Historical](/docs/VERSION/operations/api-reference.html#historical)  
+      * [Historical](/docs/VERSION/operations/api-reference.html#historical)
+  * Tuning and Recommendations
+    * [Basic Cluster Tuning](/docs/VERSION/operations/basic-cluster-tuning.html)  
+    * [General Recommendations](/docs/VERSION/operations/recommendations.html)
+    * [Performance FAQ](/docs/VERSION/operations/performance-faq.html)
+    * [JVM Best Practices](/docs/VERSION/configuration/index.html#jvm-configuration-best-practices)        
   * Tools
     * [Dump Segment Tool](/docs/VERSION/operations/dump-segment.html)
     * [Insert Segment Tool](/docs/VERSION/operations/insert-segment-to-db.html)
```

Comment:
Would you please remove this line? It was removed in https://github.com/apache/incubator-druid/pull/6911.

Reply:
I don't see insert segment tool removed. Moreover, @gianm added it back in in #7658

Reply:
I'm fine removing it as long as it is the right thing to do

Reply:
Ah I see. It looks fine to me.